### PR TITLE
Convert Meilisearch API key param to secret type

### PR DESCRIPTION
**Pull request by yamankatby**
fixes: #109

# Pull Request

## What does this PR do?
Fixes #<issue_number>

## PR checklist
Please check if your PR fulfills the following requirements:
- [ ] Does this PR fix an existing issue?
- [ ] Have you read the contributing guidelines?
- [ ] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

**Review by brunoocasali — COMMENTED** [review]
Hi @yamankatby thanks for contributing with Meilisearch ❤️ 

The maintainer of this repo @bidoubiwa is in vacation, but I think @alallema could review this ;)

**Review by alallema — APPROVED** [review]
Thanks, @yamankatby for this PR ❤️ 
LGTM! We only have to wait for @bidoubiwa to merge it

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**
